openwisp_monitoring/device/base/models.py

```diff
@@ -454,6 +455,22 @@ def handle_disabled_organization(cls, organization_id):
             status='unknown'
         )
 
+    @classmethod
+    def handle_critical_metric(cls, instance, **kwargs):
+        from openwisp_monitoring.device.settings import get_critical_device_metrics
+
+        critical_metrics = [metric['key'] for metric in get_critical_device_metrics()]
```

Comment:
once you follow the previous suggestions, you can change this to: `critical_keys = DeviceMonitoring._get_critical_metric_keys()`